### ISSUE TITLE
Quad Pattern Extract Links Actor description is hard to understand

### ISSUE THREAD

**Issue by constraintAutomaton** [comunica/comunica-feature-link-traversal#109]
#### Issue type:

- :question: Question 

____
#### Question:

The description of the actor [Quad Pattern Extract Links](https://github.com/comunica/comunica-feature-link-traversal/tree/master/packages/actor-extract-links-quad-pattern) in my opinion seems to be lacking particularly given it's similar name with [Quad Pattern Query Extract](https://github.com/comunica/comunica-feature-link-traversal/tree/master/packages/actor-extract-links-quad-pattern-query), which in my opinion make them easy to confuse. I think the description of the former should be augmented.

**Comment by rubensworks** [comment]
Indeed, could be improved. That one is also a bit harder to explain, since it depends on the query plan, and which sub-patterns are executed, which could widely vary.
